external/storm-kafka-client/src/main/java/org/apache/storm/kafka/spout/KafkaSpout.java

```diff
@@ -104,6 +104,10 @@ public KafkaSpout(KafkaSpoutConfig<K, V> kafkaSpoutConfig) {
         this.kafkaSpoutConfig = kafkaSpoutConfig;
     }
 
+    long getNumUncommittedOffsets() {
```

Comment:
Yes, the main purpose of the test is verifying commit behavior over offset gaps. We can remove this getter and the asserts on it if required.